packages/idafree.vm/tools/chocolateyinstall.ps1

```diff
@@ -30,16 +30,24 @@ try {
     Remove-Item $desktopShortcut -Force -ea 0
   }
 
+  # Download ida_launcher.exe to assist with taskbar and right click option and store it in %RAW_TOOLS_DIR%
+  # ida_launcher.exe is a custom binary that searches for the latest ida64.exe and executes it
+  $launcherName = 'ida_launcher'
+  $launcherSource = 'https://raw.githubusercontent.com/mandiant/VM-Packages/773649e54aaea62f270c8416cb480020d6475065/ida_launcher/ida_launcher.exe'
+  $launcherPath = Join-Path ${Env:RAW_TOOLS_DIR} "$launcherName.exe"
+  $launcherChecksum = "ebebf8cf01253aab0562b42128f08fdb9b4452f1f49847c51673e5610063a8b5"
+  Write-Host "[+] Downloading '$launcherSource'"
+  Get-ChocolateyWebFile -PackageName $launcherName -FileFullPath $launcherPath -Url $launcherSource -Checksum $launcherChecksum -ChecksumType "sha256"
+
+  VM-Assert-Path $launcherPath
+
+  $launcherShortcut = Join-Path ${Env:RAW_TOOLS_DIR} "$launcherName.lnk"
```

Comment:
I have just tested this PR and I think there is an important improvement we can make: what about adding the launcher link to the Tools directory as `ida` and do not create the `idafree` one?